README.md

```diff
@@ -111,6 +111,12 @@ Please refer to the [release page](https://github.com/actions/checkout/releases/
     # Default: false
     submodules: ''
 
+    # Specifies how many submodules are fetched/cloned at the same time. A positive
+    # integer allows up to that number of submodules fetched in parallel. A value of 0
+    # will give some reasonable default. If unset, it defaults to 1.
+    # Default: 1
+    submodulesFetchJobs: ''
```

Comment:
I was not sure about the name, if `submoduleFetchJobs` (aligned with `git config`) or `submodulesFetchJobs` (aligned with the other variable here)

Reply:
.github/workflows/npm-grunt.yml

Reply:
Not sure what that references